hack/update-deepcopy.sh

```diff
@@ -2,15 +2,17 @@
 
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
 
+# API_GROUP_VERSION_PACKAGES is a string of <group>/<version>.
+# The deepcopy gen needs a string of <group>:<version> so substitute slashes for colons.
+inputArg=$(echo $API_GROUP_VERSIONS | sed "s|/|:|g")
+
 verify="${VERIFY:-}"
 
 GOFLAGS="" bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy" \
   github.com/openshift/api/generated \
-  github.com/openshift/api \
-  "apiserver:v1 apps:v1 authorization:v1 build:v1 config:v1 helm:v1beta1 console:v1 console:v1alpha1 image:v1,docker10,dockerpre012 imageregistry:v1 kubecontrolplane:v1 legacyconfig:v1 cloudnetwork:v1 network:v1 networkoperator:v1 oauth:v1 openshiftcontrolplane:v1 operator:v1 operator:v1alpha1 operatorcontrolplane:v1alpha1 operatoringress:v1 osin:v1 project:v1 quota:v1 route:v1 samples:v1 security:v1 securityinternal:v1 servicecertsigner:v1alpha1 sharedresource:v1alpha1 template:v1 user:v1 machine:v1beta1 machine:v1 monitoring:v1alpha1" \
+  "${SCRIPT_ROOT}" \
+  "${inputArg}" \
```

Comment:
The old, hard-coded list is slightly different from the new, automatically generated list:
```diff
@@ -7,8 +7,10 @@
 console:v1
 console:v1alpha1
 helm:v1beta1
+image:docker10
+image:dockerpre012
 imageregistry:v1
-image:v1,docker10,dockerpre012
+image:v1
 kubecontrolplane:v1
 legacyconfig:v1
 machine:v1
@@ -32,4 +34,5 @@
 servicecertsigner:v1alpha1
 sharedresource:v1alpha1
 template:v1
+unidling:v1alpha1
 user:v1
```
I guess splitting "image:v1,docker10,dockerpre012" into separate elements is fine.  The unidling API is little used and hasn't changed since OpenShift 3; no harm in including it here.